modules/exploits/multi/browser/msfd_rce_browser.rb

```diff
@@ -0,0 +1,73 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  include Msf::Exploit::Remote::HttpServer::HTML
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Metasploit msfd Remote Code Execution',
```

Comment:
You may want to change this so it doesn't get confused with your other module (which has the same name).

Something like `Metasploit msfd Remote Code Execution Via Browser` would probably be fine